governance/xc_admin/packages/xc_admin_frontend/components/tabs/General.tsx

```diff
@@ -197,6 +197,17 @@ const General = () => {
               changes[symbol].new = { ...fileDataParsed[symbol] }
             }
           })
+          // check if any existing symbols are not in uploaded json
+          Object.keys(data).forEach((symbol) => {
+            if (!fileDataParsed[symbol]) {
+              changes[symbol] = { prev: {} }
+              changes[symbol].prev = { ...data[symbol] }
+              changes[symbol].prev.metadata = {
```

Comment:
This line doesn't do anything I think

Reply:
what do you mean? `prev.metadata` doesn't have symbol, this line adds symbol to metadata

Reply:
You'll never use symbol for a product you're about to delete

Reply:
ah yeah you're right